src/components/governance/PageProposals.vue

```diff
@@ -87,7 +88,17 @@ export default {
     proposals: {
       query() {
         /* istanbul ignore next */
-        return ProposalList(this.network)
+        return gql`
+          query proposals {
+            proposals(networkId: "${this.network}") {
```

Comment:
I believe this will not update if the networkId changes.

To fix, the query needs to be a function. You can see other examples in the PageValidators page for example.

It could be that string interpolation (` "${this.network}"`) is best/easier, but for consistency le's use a `variables` property in the `apollo` object.